combinations/hash.tsv

```diff
@@ -130,4 +130,5 @@ star=2.6.1d,samtools=1.10
 star=2.6.1d,samtools=1.10,gawk=5.1.0
 planemo=0.74.3,git=2.30.2,galaxyxml=0.4.14
 minimap2=2.18,samtools=1.12
-changeo=1.0.2,igblast=1.15.0,wget=1.20.1
\ No newline at end of file
+changeo=1.0.2,igblast=1.15.0,wget=1.20.1
+changeo=1.0.2,igblast=1.15.0,wget=1.20.1	bioconda/extended-base-image:latest	0
```

Comment:
```suggestion
changeo=1.0.2,igblast=1.15.0,wget=1.20.1	quay.io/bioconda/base-glibc-busybox-bash:2.1.0	1
```

Reply:
This should be the current base image bioconda is using. If an entry doesn't specify a build, that build is 0, so we need to set the build column to 1 here.